aten/src/ATen/native/nested/cuda/NestedTensorTransformerUtils.cpp

```diff
@@ -449,6 +472,100 @@ sdpa_nested_preprocessing(
       output_shape);
 }
 
+std::tuple<Tensor, Tensor, Tensor, Tensor, Tensor>
+sdpa_nested_preprocessing_backward(
```

Comment:
dumb question: why do we need separate preprocessing for forward and backward? I'd kinda expect the same required form for both

Reply:
we don't really need separate preprocessing it is really just to also apply the same processing on out, and grad_out